src/lightning_app/CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Added a `start_with_flow` flag to the `LightningWork` which can be disabled to prevent the work from starting at the same time as the flow ([#15591](https://github.com/Lightning-AI/lightning/pull/15591))
 
+- Added support for running Lightning App with IDE debugger ([#15590](https://github.com/Lightning-AI/lightning/pull/15590))
```

Comment:
As discussed offline, let's make it clear this only works for VSCode right now (debugpy).

Reply:
Done !